pkg/util/util_test.go

```diff
@@ -53,6 +53,63 @@ func TestUntilReturnsImmediately(t *testing.T) {
 	}
 }
 
+func TestJitterUntil(t *testing.T) {
+	ch := make(chan struct{})
+	close(ch)
```

Comment:
This test is 1:1 copy of TestUntil up to jitterFactor set to 1.0.

The idea here is to close the channel before JitterUntil (or Until) is invoked. If a channel is closed the JitterUntil (or Until) never calls function f and returns right away.